md/dev/mediquette.txt

```diff
@@ -0,0 +1,74 @@
+
+Etiquette @medic
+
+Be kind to the plants and animals.
+
+Slack
+
+  1) Try to pay attention what channel you are writing in and keep noise
+     (tangential or unrelated discussion) to a minimum.  Use threads when
+     appropriate to keep tangential discussion contained and out of the main
+     channel.   Each channel should have a purpose with a description that
+     explains appropriate usage.
+
+  2) Never delete or edit someone else's content.  If you think something is
```

Comment:
+1
I think only only admins have this ability as well, so it's kinda awkward for that too.